website/blog/how-to-migrate-an-ios-app-to-bazel.md

```diff
@@ -0,0 +1,1470 @@
+Updated project at "MastodonBazel.xcodeproj"
+```
+
+If you open the generated project
+(i.e. `xed MastodonBazel.xcodeproj`)
+and run the `Mastodon` scheme,
+it should
+build,
+install to the simulator,
+and launch successfully:
+
+![](../static/img/blog/mastodon-bazel-xcode.png)
+
+### Ensuring Bazel sees the correct Xcode version
+
+Currently there are additional steps that need to be taken in order for Bazel to correctly detect when Xcode versions change.
+I recommend reading the ["Xcode Version Selection and Invalidation"][rules_apple-xcode-management] section of the **rules_apple** docs for details on what and why.
+If you only build your project inside of Xcode,
+or with the [**rules_xcodeproj** command-line API][rules_xcodeproj-command-line-api],
+there is nothing else you need to do to manage your Xcode version with Bazel,
+because it [applies these steps for you][rules_xcodeproj-xcode-management].
+
+[rules_apple-xcode-management]: https://github.com/bazelbuild/rules_apple/blob/3.2.1/doc/common_info.md#xcode-version-selection-and-invalidation
+[rules_xcodeproj-command-line-api]: https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/1.17.0/docs/usage.md#command-line-api
+[rules_xcodeproj-xcode-management]: https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/1.17.0/xcodeproj/internal/templates/runner.sh#L148-L171
+
+## Leveraging remote caching and remote execution
+
+One of the main reasons to use Bazel,
+instead of another build system such as Xcodebuild,
+is to leverage its remote caching and execution capabilities.
+For a detailed overview on both of those capabilities,
+I recommend reading my [post on the subject][rbc-explained].
+
+For this post I'll detail the steps I would take for any Bazel migration,
+starting from no caching and ending with both caching and remote execution.
+
+[rbc-explained]: bazels-remote-caching-and-remote-execution-explained.md
+
+### Disk cache
+
+Bazel supports a form of "remote cache" that uses the local filesystem instead of a remote server.
+It's called the "disk cache" and is enabled with the [`--disk_cache`][disk_cache-flag] flag.
+
+There are pros and cons to using the disk cache:
+
+- **Pros**
+  - Lower latency than a remote cache
+  - Includes blobs that have been built locally,
```

Comment:
Perhaps link to some helpful `blobs` (or what blobs means in this case) documentation in Bazel docs.